lib/cisco_node_utils/radius_server.rb

```diff
@@ -73,7 +144,8 @@ def ==(other)
     end
 
     def auth_port
-      config_get('radius_server', 'auth-port', @name)
+      platform == :ios_xr ? @auth_port : config_get('radius_server',
+                                                    'auth-port', ip: @name)
```

Comment:
Shouldn't XR be able to use `config_get` in order to grab the proper value?  Same applies for `acct_port`